sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -67,7 +67,7 @@ export function nodeConfig({ test = false, production = false } = {}) {
 
   if (test) {
     // entry point is every test file
-    baseConfig.input = "dist-esm/test/**/*.spec.js";
+    baseConfig.input = ["dist-esm/test/public/**/*.spec.js", "dist-esm/test/internal/**/*.spec.js"];
```

Comment:
Do you need to fix the browser inputs too?

Reply:
fixed